apps/prairielearn/src/tests/sync/courseSync.test.ts

```diff
@@ -2,9 +2,12 @@ import { assert } from 'chai';
 import * as util from './util';
 import * as helperDb from '../helperDb';
 import { config } from '../../lib/config';
-import { features } from '../../lib/features';
+import { features, type FeatureName } from '../../lib/features';
 import { selectOrInsertCourseByPath } from '../../models/course';
 
+const [sampleFeature1, sampleFeature2] = features.allFeatures() as FeatureName[];
```

Comment:
Nit: is the `as FeatureName[]` necessary given that `allFeatures()` itself was changed to return `as FeatureName[]`?

Reply:
Indeed it is no longer needed. I added it before changing `allFeatures()`, forgot to remove it. Changed in cb81b0900818ffde7d8e7f3634ed9b3f43e4b334.